test/distributed/cases/dml/select/select.test

```diff
@@ -216,4 +216,12 @@ drop table if exists t1;
 create table t1 (a int(11) unsigned);
 insert into t1 values (1), (2);
 select * from t1 where a != 2;
-drop table if exists t1;
```

Comment:
please add a few more cplex queries, such as join, self join, etc.    Also some negative test, such as uuid is not a valid query, or disable save and it should fail, .etc.

Also try to create different roles, try to scan query result of a different role.   It should succeed/fail based on permission.

Reply:
Query results only provide isolation between tenants, not role level isolation. In bvt, it is not possible to add a case that can be queried by using the same query_id after the tenant is switched, because you can't get the same query_ id